HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -2190,6 +2190,7 @@ Code "Allow Cyclone Kick for Traversal" in "Gameplay" by "WasifBoomz" does "Allo
     #lib "BlackboardStatus"
 
     static int[] enableStates = { 8, 11, 46, 162 };
+    static int[] disableStates = { 77 };
```

Comment:
These need to be updated to pull from the `Sonic.StateID` enum, rather than using decimal state IDs.